waiter/config-full.edn

```diff
@@ -753,6 +753,9 @@
  ; ---------- Load Balancing ----------
 
  :work-stealing {
+                 ;; The maximum number of outstanding work-stealing offers per router:
+                 :max-work-stealing-in-flight-offers 4000
```

Comment:
We're in `:work-stealing`, so just `:max-in-flight-offers`?

Reply:
Renamed.